src/SonarScanner.MSBuild.PreProcessor/WebService/SonarCloudWebService.cs

```diff
@@ -36,9 +36,10 @@ internal class SonarCloudWebService : SonarWebService
 
         private readonly HttpClient cacheClient;
 
-        public SonarCloudWebService(IDownloader downloader, Uri serverUri, Version serverVersion, ILogger logger, HttpMessageHandler handler = null)
-            : base(downloader, serverUri, serverVersion, logger)
+        public SonarCloudWebService(IDownloader downloader, Uri serverUri, Version serverVersion, ILogger logger, string organization, HttpMessageHandler handler = null)
+            : base(downloader, serverUri, serverVersion, logger, organization)
         {
+            _ = organization ?? throw new ArgumentNullException(nameof(organization));
```

Comment:
I think we can use here `Contract.ThrowIfNullOrWhitespace` and remove the check from line 71.

Reply:
👍